{
  try {
    return operation.execute(rClass);
  }
 catch (  Exception e) {
    this.exception=e;
    throw e;
  }
}
